test/fail_compilation/test13536.d

```diff
@@ -0,0 +1,21 @@
+/*
+---
+fail_compilation/test13536.d(19): Error: field U.safeDg cannot be accessed in @safe code because it overlaps with a pointer
```

Comment:
It seems that only one of the two cases (TOKcall and TOKdotvar) is triggered (don't know which, because the msg is the same). Can you amend the test so both of the cases are triggered?